core_backend/tests/api/test_manage_content.py

```diff
@@ -96,7 +96,7 @@ def test_edit_and_retrieve_content(
         content_title: str,
         content_text: str,
         fullaccess_token: str,
-        readonly_token: str,
+        # readonly_token: str,
```

Comment:
I think we should think of new test cases. E.g.
user1 creates content1
user2 creates content2

user1 tries to read/update/delete content2 etc.

---
user1 uses user2's apikey and it works well (i think you did this in your video).

etc.

Reply:
No need to create test cases but we will need a suite of them for multi-tenancy in the end